backend/data/seeds/01_users.js

```diff
@@ -1,14 +1,24 @@
+const faker = require('faker');
 
-exports.seed = function(knex, Promise) {
-  // Deletes ALL existing entries
-  return knex('users').truncate()
-    .then(function () {
-      // Inserts seed entries
-      return knex('users').insert([
-        {user_id: 1, auth_id: 'abcd123', email: 'user1@none.com', type: 0, billing_date:null},
-        {user_id: 2, auth_id: 'bcde234', email: 'user2@none.com', type: 0, billing_date:null},
-        {user_id: 3, auth_id: 'cdef345', email: 'user3@none.com', type: 0, billing_date:null}
-      ]);
-    });
+
+const createFakeUser = () => ({
+  //auth_id: faker.finance.mask(25),
```

Comment:
make sure to remove any commits so they are not on master. You can put them back after you push to GitHub